### PR TITLE
put the try/expect into output_raster function

### PR DESCRIPTION
**Pull request by cedricserugendo**

### CODE REVIEW

**Review by neodark — APPROVED** [review]

**Comment by neodark** [comment]
@cedricserugendo : to check why some tests fail

File: cm/base/BaseCM/cm_output.py [hunk 1]
```diff
@@ -72,5 +74,9 @@ def validate(output: Dict) -> Dict:
...
     """Add a raster to the api"""
     files = {"file": (raster_name, raster_fd, "image/tiff")}
-    resp = requests.post(API_URL + "api/geofile/", files=files)
-    return resp.ok
+    try:
+        resp = requests.post(API_URL + "api/geofile/", files=files)
+        return resp.ok
+    except ConnectionError:
+        logging.error("Error during the post of the file.")
+        return False
```
Comment (melhk) [inline cm/base/BaseCM/cm_output.py:82]: The API should always return status codes, not True or False.

Comment (neodark) [reply cm/base/BaseCM/cm_output.py:82]: Yes this should be fixed as well. Thanks @melhk for pointing that out!

Comment (neodark) [reply cm/base/BaseCM/cm_output.py:82]: On a second thought, maybe we don't need status codes here as it's not an API.
This needs to be investigated more deeply:
* What does it mean when we return an error from the CM ?
* What is the procedure when everything works fine?
* What are the expected outputs for all cases ?

**Pull request merged**